src/domain/app/layout/PageLayout.tsx

```diff
@@ -20,7 +20,7 @@ import { getCmsPath } from '../routes/utils';
 import styles from './pageLayout.module.scss';
 import ProtectedPageWrapper from './ProtectedPageWrapper';
 
-export const MAIN_CONTENT_ID = 'main-content';
+export const MAIN_CONTENT_ID = 'main-content' as const;
```

Comment:
It is already an unambiguous const, so there should be no need to reintroduce it as such.

<img width="821" alt="image" src="https://github.com/City-of-Helsinki/palvelutarjotin-admin/assets/389204/0b86ccb5-52c2-4d5c-8f7b-a2ff86b15669">
https://www.typescriptlang.org/play?#code/MYewdgzgLgBAsgQQJIDkD6BhA8igKgUTzSQBEYBeGAcgFsBDASzAFpQwoBTdqgbgCg+baPGTpseQrmJlKtRizaduMOhBhCoPIA

Reply:
Learned something new today! Removed the unnecessary "as const" from string literals in the codebase.